sencha-workspace/SlateDemonstrationsStudent/app/controller/OverviewWindow.js

```diff
@@ -1,22 +1,34 @@
-Ext.define('Slate.cbl.view.student.skill.OverviewWindowController', {
-    extend: 'Ext.app.ViewController',
-    alias: 'controller.slate-cbl-student-skill-overviewwindow',
+Ext.define('SlateDemonstrationsStudent.controller.OverviewWindow', {
```

Comment:
@aliyeysides per issue #114 you were only supposed to convert the Dashboard controllers from ViewControllers to regular controllers.

Each app is focused on presenting a single dashboard, but many overview windows can be open at a time which IS where we want to use ViewControllers. Your current implementation will fail because the `skillCombo` ref will grab whatever component it finds first and always use the same one.